backend/coins/btc/account.go

```diff
@@ -873,12 +874,19 @@ func (account *Account) SpendableOutputs() []*SpendableOutput {
 		panic(err)
 	}
 	for outPoint, txOut := range utxos {
+		isChange := false
+		for _, subacc := range account.subaccounts {
```

Comment:
this is the same code as https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/81b3b257f9bad89ec94b1e6939461b2fc6960cc4/backend/coins/btc/account.go#L722-L726

would be nice to move that to a method on Account, with a unit test, so it can be reused

Reply:
Maybe it would be better to make the [AccountAddress](https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/160d1adebbabd220d83dfcffbfa0a3dda85b7a9b/backend/coins/btc/addresses/address.go#L34) _change aware_ and then use [getAddress](https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/160d1adebbabd220d83dfcffbfa0a3dda85b7a9b/backend/coins/btc/transaction.go#L218) instead of `IsChange`, here is how it could look like:
https://github.com/BitBoxSwiss/bitbox-wallet-app/commit/3b7fa1246e4cc722a908e0cc9eefedc680000dff

I think it makes sense for AccounAdress to know whether it is change or not. Often times we'll want to get some address anyways, and even if we really only care for the `IsChange` value the overhead (looking at the receiveAdresses) does not matter at all since a hash map is used for the lookup (O(1)) I think this would make the code easier to understand. 

Please lmk what you think @benma

Reply:
(added three commits after the initial backend commit, can squash them in case we want to do it like this)

Reply:
please squash